draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
               <list style="symbols">
 
-                <t>An "a=msid" line, as specified in
-                <xref target="I-D.ietf-mmusic-msid"></xref>, Section
-                2.</t>
+                <t>For each MediaStream that was associated with the
+                transceiver when it was created via addTrack or addTransceiver,
+                an "a=msid" line, as specified in
+                <xref target="I-D.ietf-mmusic-msid"></xref>, Section 2. If a
+                MediaStreamTrack is attached to the transceiver's RtpSender,
+                the "a=msid" lines MUST use that track's ID. If no
+                MediaStreamTrack is attached, a valid ID MUST be generated,
+                in the same way that the implementation generates IDs for
```

Comment:
Note that this text may not be accurate if replaceTrack has been called. There is text for subsequent offers that says that a=msid needs to remain the same, but it's not clear what should happen if replaceTrack occurs before createOffer.

In any case it would be good idea to spell this out.

Also: I think the interpreting-a-remote-description section only expects there to be a single msid attribute. We'll need to file a bug on that.